creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -603,14 +607,6 @@ class SnapbackSM {
     healthyNodes,
     replicaSetNodesToUserClockStatusesMap
   }) {
```

Comment:
i figured out what you're doing by manually tracing through the code but with a confusing diff like this, would be great if you could just comment on the PR in-line briefly what the diff is actually doing 🙏

Reply:
will keep in mind for future PRs 👍